website/content/docs/agent/template.mdx

```diff
@@ -194,36 +211,18 @@ using the certificates `validTo` field.
 This does not apply to certificates generated with `generate_lease: true`. If set
 Vault Agent template will apply the non-renewable, leased secret rules.
 
-## Auto Auth and Templating Example
-
-The following demonstrates configuring Vault Agent to template secrets using the
-AppRole Auth method:
-
-```python
-pid_file = "./pidfile"
-
-vault {
-  address = "https://127.0.0.1:8200"
-}
-
-auto_auth {
-  method {
-    type = "approle"
+When Agent's auto-auth triggers a re-authentication, due to a token expiry for
+example, it generates a new token for Agent's use. This triggers a template
+server restart, which fetched and re-renders a new set of certificates even if
```

Comment:
```suggestion
server restart, which fetches and re-renders a new set of certificates even if
```

Reply:
I also wonder if this paragraph should be called out in something like a Note text box for visibility.

Reply:
Changed it to be a Note box, though it's a bit lengthy. @taoism4504 thoughts?